packages/expo-dev-menu/ios/EXDevMenuBuildInfo.h

```diff
@@ -0,0 +1,22 @@
+//
+//  DevMenuBuildInfo.h
+//  expo-dev-menu-EXDevMenu
+//
+//  Created by andrew on 2022-01-10.
+//
+
+#import <Foundation/Foundation.h>
+#import <React/RCTBridge.h>
+#import <EXManifests/EXManifestsManifestFactory.h>
+
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXDevMenuBuildInfo : NSObject
+
++(NSDictionary *)getBuildInfoForBridge:(RCTBridge *)bridge andManifest:(id)manifest;
```

Comment:
```suggestion
+(NSDictionary *)getBuildInfoForBridge:(RCTBridge *)bridge andManifest:(NSDictionary *)manifest;
```
to match the implementation